src/inmemorystorage.ts

```diff
@@ -9,7 +8,7 @@ import {CachingLayer} from './cachinglayer-new';
  * @class
  **/
 
-export class InMemoryStorage extends CachingLayer {
+class InMemoryStorageBase extends CachingLayer {
```

Comment:
What's the `Base` suffix for?

Reply:
This is to avoid confusion on what is exported and what is to be used. Do you maybe have a better suggestion?

Reply:
I see. How about we just explain that requirement in the JSDoc comment then, and leave the name without the suffix?